Source/FullScreenPokemon.ts

```diff
@@ -8386,6 +8395,7 @@ module FullScreenPokemon {
          * @param settings   Settings used for the cutscene.
          */
         cutsceneOakIntroPokemonChoiceRivalWalksToPokemon(FSP: FullScreenPokemon, settings: any): void {
+            FSP.MapScreener.blockInputs = true;
```

Comment:
Can you move this to be with the `FSP.MenuGrapher` calls? It's weird having it up here, before `var rival`.